python/doc/user_manual/statistics_on_sample.rst

```diff
@@ -237,6 +237,7 @@ Graphical tests
     VisualTest.DrawUpperExtremalDependenceFunction
     VisualTest.DrawLowerTailDependenceFunction
     VisualTest.DrawLowerExtremalDependenceFunction
+    VisualTest.DrawInsideOutside
```

Comment:
pyplost/DrawInsideOutside.py is missing

Reply:
hello @josephmure do you think you can take of this for the release ?

Reply:
can you try to rebase ?

Reply:
done